hw1/02-palindrome.js

```diff
@@ -1,3 +1,28 @@
-const elem = document.querySelector('input');
 
-elem.addEventListener('input', handleInput);
+// // Attach an event listener to the form element to prevent default submission and check palindrome
+document.getElementById('palindromeForm').addEventListener('submit', (event) => {
```

Comment:
Regarding your event listener, the submit event seems to be called when you submit the form. That should mean it should only check if it is a palindrome, not as you write the word/etc.

Reply:
Yes, it checks if it is a palindrome when you submit the form.

Reply:
Just wanting to make sure is this the only comment visible. Because the other comments are not showing.